content/coronavirus_worker_page.yml

```diff
@@ -0,0 +1,89 @@
+content:
+  title: "Coronavirus (COVID-19): Work and financial support"
+  meta_description: ""
+  page_title: "Work and financial support"
+  header_section:
+    pretext: Coronavirus (COVID-19) support is available
+    list:
+      - "some workplaces are starting to open - with social distancing"
+      - "plan your journey to avoid crowds"
+      - "financial support is available if you cannot work"
```

Comment:
The quotes are not needed here either and in other plain text elements further down.